python/getCustomAttributes.py

```diff
@@ -50,7 +50,7 @@ def getCustomAttributes(data: dict) -> dict:
 
     return attributes
 
-def getCustomAttributes(data: dict) -> dict:
+def getCustomAttributesEnd(data: dict) -> dict:
```

Comment:
Both functions were mistakenly called `getCustomAttributes()` but of course we want two different functions! This mirrors the name used in the Javascript version of this file.